src/main/bvmui/dist/index.html

```diff
@@ -1 +1 @@
-<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><title>BK - Visual Manager</title><link rel="shortcut icon" href=favicon.ico><link href=/css/app.9f6a864a.css rel=preload as=style><link href=/js/app.cec374e4.js rel=preload as=script><link href=/js/chunk-vendors.8c726444.js rel=preload as=script><link href=/css/app.9f6a864a.css rel=stylesheet></head><body><noscript><strong>We're sorry but Bookkeeper Visual Manager doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=/js/chunk-vendors.8c726444.js></script><script src=/js/app.cec374e4.js></script></body></html>
```

Comment:
I think we can drop the dist folder and add it to .gitignore

Reply:
there was a typo in .gitignore
fixed